python/src/tests/tasks/test_embedding.py

```diff
@@ -80,4 +79,3 @@ def test_tsne_edits_object_appropriately(self):
     def test_throws_on_invalid_embedding_type(self):
         with pytest.raises(Exception):
             ComputeEmbedding(self).compute("definitelynotavalidembedding")
```

Comment:
Does not look like we are testing what we say we are testing 🤔 . I think that we need to create something like
```
            "body": {
                "name": "GetEmbedding",
                "type": "definitelynotavalidembedding",
            }
```

Reply:
Actually, I think in this case the title of the test is wrong, but the actual test contents are correct. If you look at this line: https://github.com/biomage-ltd/worker/blob/master/python/src/tasks/embedding.py#L25 there is no validation of which task type we are doing. Validation of task name happens here: https://github.com/biomage-ltd/worker/blob/master/python/src/tasks/tasks.py#L57 So this test basically checks what will happen if there is no `task_def` defined in `self` of the class.

Reply:
so I think just renaming the title of the test to should work